### DISCUSSION TITLE
[Feature] Have a info and an advanced info on mobile for admins

### DISCUSSION THREAD

**Discussion by noki2000** [immich-app/immich#9631] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello!

Currently on the mobile version of the administrator, tapping on the only info icon present in the image view opens an advanced info.
It shows informations like this:
![photo1716289467](https://github.com/immich-app/immich/assets/161160694/8843b4eb-2f50-469e-a4a5-9b73307270af)

For other users the info button presents a more user friendly option, like the one present on the web:
<img width="1440" alt="Bildschirmfoto 2024-05-21 um 13 08 10" src="https://github.com/immich-app/immich/assets/161160694/493d93d6-a451-4859-b5a6-2e037a2ca92f">

While the info button of the admin presents lots of useful data, it is neither user-friendly nor does it display the same informations as the user version. The user version presents the geolocation and the face recognition information, not displayed in the advanced info of the admin.

Therefore I would suggest, for the mobile, to have the same info icon for both user and admin to point to the info now visible for the user. Then add an "advanced information" link, from within the info button, to open the current advanced info.

The advanced info might be, in my opinion, also visible for the user. Maybe it could then be presented in a prettier way.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
The advanced information is a debug toggle in the app's settings that you turned on. It's unrelated to being an admin or not.

**Reply by noki2000** [reply]
Thanks @bo0tzz :)
